<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>781</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>First Queue Number: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="textNumberFirst"/>
           </item>
           <item>
            <widget class="QPushButton" name="buttonGenerateFirst">
             <property name="text">
              <string>Genearate First</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Second Queue Number: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="textNumberSecond"/>
           </item>
           <item>
            <widget class="QPushButton" name="buttonGenerateSecond">
             <property name="text">
              <string>Genearate Second</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="buttonMinFirst">
           <property name="text">
            <string>Push Min In First</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonSwap">
           <property name="text">
            <string>Swap</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonMinSecond">
           <property name="text">
            <string>Push Min In Second</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QListWidget" name="listFirst"/>
       </item>
       <item row="0" column="1">
        <widget class="QListWidget" name="listSecond"/>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="buttonAddFirst">
             <property name="text">
              <string>Add To First</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonRemoveFirst">
             <property name="text">
              <string>Remove From First</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Input Integer: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="textValue"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="buttonAddSecond">
             <property name="text">
              <string>Add To Second</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonRemoveSecond">
             <property name="text">
              <string>Remove From Second</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
